articles/quickstart/backend/aspnet-core-webapi-2/03-troubleshooting.md

```diff
@@ -45,8 +45,8 @@ You can inspect a JWT with the [JWT.io](https://jwt.io/) website. Use the debugg
 
 The screenshot below shows the following information:
 * The token is signed with the RS256 algorithm
-* The issuer of the token is https://jerrie.auth0.com/
-* The audience of the token is https://rs256.test.api
+* The issuer of the token is `https://jerrie.auth0.com/`
```

Comment:
I do not want to block the PR om this, but want to ask ...  What do we do with these kind of links, given they are not part of the SDK anymore ? I know these are merely here as a matter of explanation but should we consider updating this to something else, eventually? This probably has implications on the samples as well tho.

Reply:
I've renamed the issuer value to `https://example.auth0.com` throughout.